frontend/src/store/store.js

```diff
@@ -68,6 +68,14 @@ export const store = new Vuex.Store({
     description: '',
     token: null,
     newsletterState: null,
+    community_name: 'Community 1',
+    community: {
```

Comment:
We want this from the .env or I am mistaken here?

Reply:
yes, this should probably be in the env file, but since i don't have any exact details and this was only touched on in the conversation yesterday ... i'm waiting for an exact specification. how and where?